tests/baselines/reference/privateNamesUnique-3.errors.txt

```diff
@@ -0,0 +1,29 @@
+tests/cases/conformance/classes/members/privateNames/privateNamesUnique-3.ts(4,25): error TS2339: Property '#x' does not exist on type 'Child'.
+tests/cases/conformance/classes/members/privateNames/privateNamesUnique-3.ts(4,25): error TS18014: The property '#x' cannot be accessed on type 'Child' within this class because it is shadowed by another private identifier with the same spelling.
+
+
+==== tests/cases/conformance/classes/members/privateNames/privateNamesUnique-3.ts (2 errors) ====
+    class Parent {
+        #x;
+        copy(child: Child) {
+            this.#x = child.#x; // OK (Sub has Super's #x)
+                            ~~
+!!! error TS2339: Property '#x' does not exist on type 'Child'.
+                            ~~
+!!! error TS18014: The property '#x' cannot be accessed on type 'Child' within this class because it is shadowed by another private identifier with the same spelling.

```

Comment:
the error here doesn't agree with the OK comment. (I think the comment is correct.)

Reply:
The comment is correct. Private names are not "inherited", per se, but the private names of the superclass are initialized as soon as you call `super()` (either explicitly or implicitly in a class without a constructor).

Tested and verified in Chrome Canary:
![private names in chrome canary](https://user-images.githubusercontent.com/3902892/61418048-1add5700-a8ae-11e9-88c8-bb36cbbcb1e9.png)

Reply:
Its important to remember the `#x` you're accessing in `Parent.copy` is not `Child`'s `#x`, but `Parent`'s `#x`.

Here's another example to illustrate this case:
```ts
class Parent {
  #x: number = 1;
  copy(child: Child) {
    const x: number = child.#x; // ok, because `#x` in `Parent` is type `number`
  }
}
class Child extends Parent {
  #x: string = "hi";
}
```